### ISSUE TITLE
BUG: Safari - Focus outline missing on radio and checkbox elements

### ISSUE THREAD

**Issue by esteinborn** [ITS-HCD/nysds#108]
Latest MacOS, Latest Safari:

![checkbox-active](https://github.com/user-attachments/assets/e147eb39-4d36-41e3-be33-b823ea2e7a27)
